### PR TITLE
[Documentation] Updates to Getting Started

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Created PR to highlight specific changes that update instructions for getting a local environment setup.

#### Pull Request Creator Checklist

- [ ] ~~This PR has an associated issue or issues.~~
- [ ] ~~The associated issue(s) are linked above.~~
- [ ] ~~This PR meets all acceptance criteria for those issues.~~
- [ ] ~~This PR and linked issue(s) are adequately documented~~
- [ ] ~~This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.~~
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: README2.md [hunk 1]
```diff
@@ -15,13 +15,51 @@ The new web-based QMR application will allow CMS to access data submitted by the
...
-If you haven't already, you need to first create an EUA user ID by visiting this site: https://eua.cms.gov/iam/im/pri/
+## Register an EUA Account
 
-## Eua
+In order to make changes against this repo and interact with associated services, you will need an EUA account with appropriate job codes. Create an EUA user ID here: [https://eua.cms.gov/iam/im/pri/](https://eua.cms.gov/iam/im/pri/).
 
-EUA is the first step to getting started with the application. Access to everything else starts here and can take days to weeks to process so it's best to request access as early as possible.
+EUA access requests may take days or weeks to process, but local development setup can continue.
 
-### List of EUA access codes needed for QMR and MacPRO projects in general
```
Comment (jphechter) [inline README2.md:24]: I feel like this should be documented somewhere else. This has less to do with the actual codebase and it's functionality and more to do with a contract or job roles. There's a case to say "add Github/AWS job code" but again, I don't think that belongs in the repo.

**Comment by jphechter** [comment]
> Could you add a section about setting up your local ENV? Right now we don't have any sensitive information in our ENV so you could straight up copy your main ENV, and your UI-service env in a code block for people to copy themselves

@abelzer212 we do have `.env_example` tracked in version control and the following as part of the `dev` script.
```bash
# if .env doesn't exist, copy .env_example there
if [ ! -f .env ]; then 
	cp .env_example .env
fi
```

I'm not opposed to discussing the `.env` file in more detail, but maybe that could be another section of the documentation. Thoughts?

File: README2.md [hunk 2]
```diff
@@ -15,13 +15,51 @@ The new web-based QMR application will allow CMS to access data submitted by the
...
+
+1. Install all other node packages.
+   ```
+   yarn install  # can be skipped, will run automatically in dev script
+   ```
+1. Run the application.
+   ```
+   ./dev local
+   ```
+
```
[the remark this answers is not in the document]
Comment (abelzer212) [reply README2.md:59]: Actually just an internal hyperlink to the AWS section will work. I'll add a small section to the AWS portion on how to find test credentials

File: README2.md [hunk 3]
```diff
@@ -15,13 +15,51 @@ The new web-based QMR application will allow CMS to access data submitted by the
...
+EUA access requests may take days or weeks to process, but local development setup can continue.
 
-### List of EUA access codes needed for QMR and MacPRO projects in general
+## Local Development Setup
+
+The following are pre-requesites for local development.
+
+1. Install [Node](https://nodejs.org/en/download/)
+1. Install [Node Version Manager (NVM)](https://github.com/nvm-sh/nvm#installing-and-updating)
+   - A specific version of Node is enforced and specified in the file `.nvmrc`. This version matches the Lambda runtime.
```
[the remark this answers is not in the document]
Comment (jphechter) [reply README2.md:30]: Included.

File: README2.md (hunk 1 above)
Comment (abelzer212) [reply README2.md:24]: I would just include it all though and we can make another story to move relevant documents and info into a CMS confluence page.

**Pull request merged**